README.md

```diff
@@ -5,11 +5,53 @@ The case of integration tests can be used for testing functionality of JasperRep
 
 Table of Contents
 ------------------
-1. [Configuration](#configuration).
+1. [Running application](#running-application).
+2. [File configuration](#file-configuration).
+
+Running application
+-------------
+To start working with the test case you should have installed `Oracle/Sun Java JDK  1.6 or 1.7`, `Apache Maven 3.x` and `Git` (or anther tool that allow you clone the project from GitHub). Follow next steps to run the tests:
+1. Clone the repository to your local computer using Git command: 
+```java
+git clone https://github.com/Jaspersoft/jrs-rest-java-client-tests.git
+```
+You can skip this step and download source code directly from the main page of repository `https://github.com/Jaspersoft/jrs-rest-java-client-tests`.
+2. Run application form command line with default properties:
+```java
+mvn test
+```
+You can specify variables that are used as properties for application in command line:
+```java
+mvn test -DvariableName="variableName"
+```
+for example:
+```java
+test -Djrs-client.version=6.1.4 -Dusername="jasperadmin" -Dpassword="jasperadmin"
+```
+The application properties have been predefined but you can override them with command line (or Run configuration your IDE).
+Full list of variables that can be overridden:
+
+| Variable  | Default value  |
+| :--- | :--- |
+|test.properties.dir|src/main/resources|
```

Comment:
I would rename 'test.properties.dir' to 'test.properties.path' with ability to specify full path to property file including file name.